roles/centos7/opensearch/tasks/security.yml

```diff
@@ -103,6 +103,10 @@
     state: restarted
     enabled: yes
 
+- name: Pause for 3 seconds to provide sometime for OpenSearch start
+  pause:
+    seconds: 3
```

Comment:
Can we wait for OpenSearch to start rather than this pause, different environments will have different startup times and this will be prone to failure.

Reply:
@saravanan30erd ^^

Reply:
will create a followup issue to change this.